rpcs3/Emu/Memory/vm.h

```diff
@@ -16,6 +16,7 @@ namespace vm
 	{
 		main,
 		user_space,
+		rsx_context,
```

Comment:
the code very much depends currently on being able to get the 0x40000000 allocation for context, unless there is a better way to do it that im missing?